snap/local/bin/source-env-file.sh

```diff
@@ -1,16 +1,4 @@
-#!/bin/bash -e
-
-EDGEX_STARTUP_DURATION=$(snapctl get startup-duration)
-
-if [ -n "$EDGEX_STARTUP_DURATION" ]; then
-  export EDGEX_STARTUP_DURATION
-fi
-
-EDGEX_STARTUP_INTERVAL=$(snapctl get startup-interval)
-
-if [ -n "$EDGEX_STARTUP_INTERVAL" ]; then
-  export EDGEX_STARTUP_INTERVAL
-fi
+#!/bin/bash -ex
```

Comment:
I think if xtrace is enabled, the commands will appear in syslogs without the appropriate component name or tag. The reader can't know which application they come from.